lib/jsonite/lets_proxy.rb

```diff
@@ -0,0 +1,23 @@
+class Jsonite
+  class LetsProxy < BasicObject
+
+    undef_method :==, :!, :!=
+
+    def initialize object, context, lets = {}
+      @__object, @__context, @__lets = object, context, lets
+
+      @__memoized = ::Hash.new do |memoized, name|
+        memoized[name] = instance_exec @__context, &@__lets.fetch(name)
+      end
+    end
+
+    def method_missing name, *args, &block
+      if @__lets.key?(name.to_s)
+        @__memoized[name.to_s]
```

Comment:
Oh right. I guess it's safer to be consistent with strings. We might want to benchmark symbols vs. strings with a large dataset and see if it matters.